tests/handlebars-attr-node/brackets.hbs

```diff
@@ -0,0 +1,2 @@
+<div title='My "title"'></div>
+<div title="My other 'title'"></div>
```

Comment:
Let's add here:
- `<div title='My title'></div>`
- `<div title="My title"></div>`

for consistently testing and we can merge :+1:

Reply:
👍
I've added these two cases in a last commit.